webrtc-stats.html

```diff
@@ -1775,6 +1777,43 @@
                   mapped to 127.
                 </p>
               </dd>
+              </dd>
+              <dt>
+                <dfn><code>totalAudioEnergy</code></dfn> of type <span class=
+                "idlMemberType"><a>double</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for audio. This value MUST be computed as follows:
+                  for each audio sample sent/received for this object (and
+                  counted by <code><a>totalSamplesSent</a></code> or
+                  <code><a>totalSamplesReceived</a></code>), add the square of
+                  the sample's value divided by the highest-intensity encodable
+                  value, multiplied by the duration of the sample in seconds.
+                </p>
+                <p>
+                  This can be used to obtain a root mean square (RMS) value
+                  that uses the same units as <code><a>audioLevel</a></code>,
+                  using the formula
+                  <code>Math.sqrt(totalAudioEnergy/totalSamplesDuration)</code>.
+                  This calculation can also be performed using the
+                  differences between the values of two different
+                  <code>getStats()</code> calls, in order to compute the
+                  average audio level over any desired time interval.
+                </p>
+                <p>
+                  For example, if a 10ms packet of audio is received with an
+                  RMS of 0.5 (out of 1.0), this should add <code>0.5 * 0.5 *
+                  0.01 = 0.0025</code> to <code>totalAudioEnergy</code>. If
+                  another 10ms packet with an RMS of 0.1 is received, this should
+                  similarly add <code>0.0001</code> to
+                  <code>totalAudioEnergy</code>. Then,
+                  <code>Math.sqrt(totalAudioEnergy/totalSamplesDuration)</code>
+                  becomes <code>Math.sqrt(0.0026/0.02) = 0.36</code>, which is
+                  the same value that would be obtained by doing an RMS
+                  calculation over the contiguous 20ms segment of audio.
```

Comment:
nit: Can you change this example to include two getStats calls so that people don't get confused about where to do the sqrt?